library/src/scala/tasty/reflect/SourceCodePrinter.scala

```diff
@@ -1409,11 +1409,26 @@ class SourceCodePrinter[R <: Reflection & Singleton](val tasty: R)(syntaxHighlig
     private def escapedString(str: String): String = str flatMap escapedChar
   }
 
+  private[this] val names = collection.mutable.Map.empty[Symbol, String]
+  private[this] val namesIndex = collection.mutable.Map.empty[String, Int]
```

Comment:
For later optimization: instead of generating global unique names, it's sufficient to maintain unique names in the static scope.

Reply:
Yes. I was thinking of leaving it as an external contribution for someone.